packages/backend/src/indexers/EventIndexer.ts

```diff
@@ -128,8 +160,7 @@ export class EventIndexer extends ChildIndexer {
   }
 
   protected override async invalidate(targetHeight: number): Promise<number> {
-    const blockNumber = Math.max(targetHeight, this.startBlock)
-    await this.eventRepository.deleteAfter(blockNumber, this.chainId)
+    await this.eventRepository.deleteAfter(targetHeight, this.chainId)
```

Comment:
What's up with removing start block ❔

Reply:
The problem was that we would not remove the startBlock record when trying to invalidate to 0. This was a bug. Also, we can actually invalidate to 0, as the update will start from `startBlock` anyway, and this way we're sure that everything is clean.